tests/pipes/test_data/mock_data.py

```diff
@@ -43,7 +43,20 @@
     "statements": [],
     "resourceUrls": []
 }
+REFERENCE_LINE = {
+    'statement': {},
+    'itemId': ITEM_ID,
+    'reference': {'referenceMetadata': {},
+                  'extractedData': []}}
 
+EXAMPLE_LINE = {
+    **REFERENCE_LINE,
+    'statement': STATEMENT_BLOB,
+    'reference': {
+        'referenceMetadata': {WHITELISTED_EXT_ID: 'fooid'},
+        'extractedData': ['foo', 'bar']
+    }
+}
```

Comment:
Fabulous! I'm happy to see that this file and the constants in it is also useful for other devs

Reply:
Thank you for doing it